src/main/java/org/jenkinsci/plugins/durabletask/executors/OnceRetentionStrategy.java

```diff
@@ -54,6 +57,23 @@
      */
     public OnceRetentionStrategy(int idleMinutes) {
         super(idleMinutes);
```

Comment:
This patch basically makes it pointless to extend `CloudRetentionStrategy` at all, but that is a subject for a future cleanup refactoring. :+1: